assets/js/modules/search-console/components/dashboard/DashboardImpressionsWidget.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * DashboardImpressionsWidget component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { extractSearchConsoleDashboardData } from '../../util';
+import { extractForSparkline, getSiteKitAdminURL } from '../../../../util';
+import { trackEvent } from '../../../../util/tracking';
+import whenActive from '../../../../util/when-active';
+import DataBlock from '../../../../components/data-block';
+import Sparkline from '../../../../components/sparkline';
+import PreviewBlock from '../../../../components/preview-block';
+import getDataErrorComponent from '../../../../components/notifications/data-error';
+import noDataComponent from '../../../../components/notifications/nodata';
```

Comment:
See above.

Reply:
Updated

Reply:
You beat me to it by 3 minutes, I was gonna quickly apply the same change 🤣